### ISSUE TITLE
To be able to search a city weather by an input 🚀

### ISSUE THREAD

**Issue by jennifertakagi** [jennifertakagi/weather-app#1]

**Comment by gabrielsanttana** [comment]
Hi, Jennifer! Is it open for contributions? If so, I'd like to contribute! 🙂

**Comment by jennifertakagi** [comment]
> Hi, Jennifer! Is it open for contributions? If so, I'd like to contribute! 🙂

Yes @gabrielsanttana!! I will draw a simple mock with the initial idea this week.
I'll let you know when I attach here :)

Thanks for the interest :)))
